Jenkinsfile

```diff
@@ -36,12 +34,10 @@ timeout(time: 15, unit: 'MINUTES') {
 	}
 }
 
-def createBuildTask(label) {
+def createBuildTask(jvmToTest) {
 	return {
 		node('default') {
-			withEnv(["TARGET_COMPATIBILITY=${label}"]) {
-				echo "Build for Java ${label}"
-
+			withEnv(["JVM_TO_TEST=${jvmToTest}"]) {
```

Comment:
Also here, as this is a list of JVMs, this should probably be named JVMS_TO_TEST.